src/corg/developer-environment-setup/installing-jdk-and-related-tools.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Installing JDK and related tools
+labels:
+    - java
+    - jdk
+    - eclipse
+    - intellij
+    - idea
+    - maven
+---
+
+## Installing JDK
+
+Nuxeo Platform is a Java based solution. Developing Nuxeo Platform requires JDK 8 installed on your computer. You can
+download the [latest JDK 8 version from Oracle](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html),
+or choose your preferred way to download a JDK.
+
+Once installed, verify the Java compiler version in terminal. The result should be in Java 8:
+
+```
+$ javac -version
+javac 1.8.0_131
+```
+
+## Installing Related Tools
+
+You need to install Maven, a project management tool. You also need to install your preferred IDE, such as Eclipse or IntelliJ.
+
+### Maven
```

Comment:
No exactly. I agree that the Maven usage has been well described in <https://doc.nuxeo.com/corg/maven-usage/>, but when you read the doc in order, you'll find that the page « §2.3 Maven Usage » comes after the page « §1.3 Installing JDK and related tools ».

1. Developer Environment Setup
2. Development Tools and Process
3. Coding and Design Guidelines
4. Working on Nuxeo sources

The critical point here is a user cannot get build the Nuxeo source if they don't have Maven installed.